addons/binding/org.openhab.binding.homepilot/src/main/java/org/openhab/binding/homepilot/handler/HomePilotBridgeHandler.java

```diff
@@ -0,0 +1,137 @@
+package org.openhab.binding.homepilot.handler;
+
+import java.util.Hashtable;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.config.discovery.DiscoveryService;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.State;
+import org.openhab.binding.homepilot.discovery.HomePilotDeviceDiscoveryService;
+import org.openhab.binding.homepilot.internal.HomePilotConfig;
+import org.openhab.binding.homepilot.internal.HomePilotGateway;
+import org.openhab.binding.homepilot.internal.HomePilotGatewayFactory;
+import org.osgi.framework.ServiceRegistration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HomePilotBridgeHandler extends BaseBridgeHandler {
+
+    private static final Logger logger = LoggerFactory.getLogger(HomePilotBridgeHandler.class);
```

Comment:
Logger should NOT be static according to the guidelines. [[source](http://docs.openhab.org/developers/development/guidelines.html#e-logging)]